underscore.js

```diff
@@ -647,6 +647,23 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count || 1;
```

Comment:
I think the current behavior is correct if you're chunk is less than 1 there is no chunk and a shallow clone is returned.
Think of it like if you had a pie and you cut it into 0 slices you still have a whole pie.